pipeline/config-dev.groovy

```diff
@@ -179,10 +182,39 @@ app {
                             'CACHE_REDIS_HOST': "mds-redis${vars.deployment.suffix}",
                             'ELASTIC_ENABLED': "${vars.deployment.elastic_enabled_core}",
                             'ELASTIC_SERVICE_NAME': "${vars.deployment.elastic_service_name}",
-                            'DOCUMENT_CAPACITY':"${vars.DOCUMENT_PVC_SIZE}",
                             'ENVIRONMENT_NAME':"${app.deployment.env.name}",
                             'API_URL': "https://${vars.modules.'mds-nginx'.HOST_CORE}${vars.modules.'mds-nginx'.PATH}/api",
                             'NRIS_API_URL': "${vars.modules.'mds-nris-backend'.HOST}${vars.modules.'mds-nris-backend'.PATH}",
+                            'DOCUMENT_MANAGER_URL': "${vars.modules.'mds-docman-backend'.HOST}${vars.modules.'mds-docman-backend'.PATH}",
+                    ]
+                ],
+                [
+                    'file':'microservices/document_manager/openshift/_python36_docman.dc.json',
+                    'params':[
+                            'NAME':"mds-docman-backend",
+                            'SUFFIX': "${vars.deployment.suffix}",
+                            'VERSION':"${app.deployment.version}",
+                            'CPU_REQUEST':"${vars.resources.python_lite.cpu_request}",
+                            'CPU_LIMIT':"${vars.resources.python_lite.cpu_limit}",
+                            'MEMORY_REQUEST':"${vars.resources.python_lite.memory_request}",
+                            'MEMORY_LIMIT':"${vars.resources.python_lite.memory_limit}",
```

Comment:
Should there be a separate set of resource block for document manager? Does it need same resources as the nris backend? 
Also does it behave correctly with multiple pods?

Reply:
I hammered dev with 3 pods and a mixture of large and small files and two browers at once and saw no issues. I think this will behave correctly (The TUS upload standard is a serial protocol).